internal/controller/etcdcopybackupstask/reconciler.go

```diff
@@ -483,18 +492,30 @@ func createVolumeMountsFromStore(store *druidv1alpha1.StoreSpec, provider, volum
 		}
 	case utils.GCS:
 		volumeMounts = append(volumeMounts, corev1.VolumeMount{
-			Name:      getVolumeNamePrefix(volumeMountPrefix) + "etcd-backup",
-			MountPath: "/var/." + getVolumeNamePrefix(volumeMountPrefix) + "gcp/",
+			Name:      getVolumeNamePrefix(volumeMountPrefix) + common.ProviderBackupSecretVolumeName,
+			MountPath: getGCSSecretVolumeMountPathWithPrefixAndSuffix(getVolumeNamePrefix(volumeMountPrefix), "/"),
 		})
 	case utils.S3, utils.ABS, utils.Swift, utils.OCS, utils.OSS:
 		volumeMounts = append(volumeMounts, corev1.VolumeMount{
-			Name:      getVolumeNamePrefix(volumeMountPrefix) + "etcd-backup",
-			MountPath: "/var/" + getVolumeNamePrefix(volumeMountPrefix) + "etcd-backup/",
+			Name:      getVolumeNamePrefix(volumeMountPrefix) + common.ProviderBackupSecretVolumeName,
+			MountPath: getNonGCSSecretVolumeMountPathWithPrefixAndSuffix(volumeMountPrefix, "/"),
 		})
 	}
 	return
 }
 
+func getNonGCSSecretVolumeMountPathWithPrefixAndSuffix(volumePrefix, suffix string) string {
```

Comment:
```suggestion
func getNonGCSSecretVolumeMountPathWithPrefixAndSuffix(volumePrefix, volumeSuffix string) string {
```
I guess both should have a volume in their name or none